src/test/java/com/angelbroking/smartapi/smartstream/SmartStreamTickerTest.java

```diff
@@ -61,6 +61,25 @@ void testSmartStreamTicketLTP() throws WebSocketException, InterruptedException
 
 	}
 
+	@Test
+	void testSmartStreamTicket_WithInitialDelayAndPeriod() throws WebSocketException, InterruptedException {
+		try {
+			SmartStreamTicker ticker = new SmartStreamTicker(clientID, feedToken, new SmartStreamListenerImpl(), 6000, 6000);
+			ticker.connect();
+			ticker.subscribe(SmartStreamSubsMode.SNAP_QUOTE, getTokens());
+//			ticker.subscribe(SmartStreamSubsMode.SNAP_QUOTE, getTokens());
```

Comment:
Remove unused code

Reply:
This code is not unused, it is used in test case where we implement the SmartApiTicker websocket.